python-backend/app/config.py

```diff
@@ -46,6 +49,12 @@ class TestConfig(Config):
     JWT_OIDC_TEST_CLIENT_SECRET = "test_secret"
     JWT_OIDC_TEST_ISSUER = "test_issuer"
     # Dummy Private Keys for testing purposes, can replace these keys with any other generated key.
+
+    #Flask-uploads configs
+    UPLOADED_DOCUMENT_DEST = os.environ.get('UPLOADED_DOCUMENT_DEST', '/app/document_uploads')
+    #100MB file limit
+    FILE_BYTE_LIMIT = 100 * 1024 * 1024
```

Comment:
Duplicate code?

Reply:
No one is for the test config and one is the normal config, unless I'm missing something and the test app will pull the normal config.

Reply:
I swapped it around so that there is only one calculation now instead of two to go from MB to bytes and back again. Now it stays as MB and then for the configuration does one calculation to get the bytes.

Reply:
@JustinMacaulay @NWCalvank The TestConfig class subclasses from the Config class. So anything declared in Config is inherited by TestConfig as well.
You do not need to re-declare the variables